vdb-bench-assembly/plugins/vdb-bench-core/TranslatorSelectionService.js

```diff
@@ -23,7 +23,26 @@
         tran.deploymentTranslatorName = null;
         tran.deploymentSuccess = false;
         tran.deploymentMessage = null;
-
+        
+        // Translator image mappings
+        var imageLinkMap = {
+            "accumulo" : "plugins/vdb-bench-core/content/img/Accumulo_dv_logos_70x40.png",
+            "file" : "plugins/vdb-bench-core/content/img/FileSystem_dv_logos_70x40.png",
+            "google-spreadsheet" : "plugins/vdb-bench-core/content/img/Google_dv_logos_70x40.png",
+            "h2" : "plugins/vdb-bench-core/content/img/H2_dv_logos_70x40.png",
+            "infinispan" : "plugins/vdb-bench-core/content/img/Infinispan_dv_logos_70x40.png",
+            "ldap" : "plugins/vdb-bench-core/content/img/LDAP_dv_logos_70x40.png",
+            "modeshape" : "plugins/vdb-bench-core/content/img/ModeShape_dv_logos_70x40.png",
+            "mongodb" : "plugins/vdb-bench-core/content/img/MongoDB_dv_logos_70x40.png",
+            "mysql" : "plugins/vdb-bench-core/content/img/MySQL_dv_logos_70x40.png",
+            "mysql5" : "plugins/vdb-bench-core/content/img/MySQL_dv_logos_70x40.png",
+            "postgresql" : "plugins/vdb-bench-core/content/img/PostgresSql_dv_logos_70x40.png",
+            "salesforce" : "plugins/vdb-bench-core/content/img/Salesforce_dv_logos_70x40.png",
+            "salesforce-34" : "plugins/vdb-bench-core/content/img/Salesforce_dv_logos_70x40.png",
+            "teiid" : "plugins/vdb-bench-core/content/img/Teiid_dv_logos_70x40.png",
+            "ws" : "plugins/vdb-bench-core/content/img/WebService_dv_logos_70x40.png"
+        };
```

Comment:
Can this be made more maintainable by making 'plugins/vdb-bench-core/content/img/' a single constant and deriving the image names from the key, eg. teiid -> Teiid_dv_logos_70x40.png. Albeit couple might have to be renamed, eg. ws.